tree-construction/tests11.dat

```diff
@@ -272,6 +272,62 @@
 |       zoomandpan=""
 
 #data
+<!DOCTYPE html><body><svg contentScriptType='' contentStyleType=''externalResourcesRequired='' filterRes=''></svg>
+#errors
+#document
+| <!DOCTYPE html>
+| <html>
+|   <head>
+|   <body>
+|     <svg svg>
+|       contentscripttype=""
+|       contentstyletype=""
+|       externalresourcesrequired=""
+|       filterrest=""
```

Comment:
`filterres` isn't in the list of SVG attributes to be adjusted any longer. Also, you mistyped it anyway.